modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-ipsec-north-south-enable_{context}"]
+= Enabling IPsec encryption for external IPsec endpoints
+
+// This procedure requests installing Butane to prepare the machine config
+
+As a cluster administrator, you can enable IPsec encryption between the cluster and external IPsec endpoints. Because this procedure uses Butane to create machine configs, you must have the `butane` command installed.
+
+[NOTE]
+====
+After you apply the machine config, the Machine Config Operator reboots affected nodes in your cluster to rollout the new machine config.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* You have reduced the size of your cluster MTU by `46` bytes to allow for the overhead of the IPsec ESP header.
+* You have installed the `butane` utility.
+* You have an existing `cacert.p12` certificate bundle to copy to cluster hosts.
+
+.Procedure
+
+As a cluster administrator, you can enable IPsec support external for IPsec endpoints.
+
+. Create an IPsec configuration file named `ipsec-endpoint-config.conf`. The configuration is consumed in the next step. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/configuring-a-vpn-with-ipsec_securing-networks#configuring-a-vpn-with-ipsec_securing-networks[Configuring a VPN with IPsec].
+
+. Provide your certificate bundle in a file named `cacert.p12`. This file is imported as part of the Butane configuration in subsequent steps.
```

Comment:
@jab-rh we are not using `cacert.p12` here but `ca.pem` in butane example. As its not advisable to put the CA private key in MC so ca.pem is something we advise to customers.

Reply:
we need cert in p12 format for the `left_server` (OCP)  because we also need the private key

Reply:
@anuragthehatter, how would I describe each of these files:

* `left_server.p12`: This contains what? Why?
* `ca.pem`: This contains what? Why?

Thanks!

Reply:
@jab-rh I guess we are good. You have already taken care of it via

```
left_server.p1`: The certificate bundle for the IPsec endpoints
ca.pem`: The certificate authority that you signed your certificates with
```